pkg/executor/executortype/poolmgr/gp.go

```diff
@@ -722,7 +735,7 @@ func (gp *GenericPool) getFuncSvc(ctx context.Context, fn *fv1.Function) (*fscac
 }
 
 // destroys the pool -- the deployment, replicaset and pods
-func (gp *GenericPool) destroy() error {
+func (gp *GenericPool) destroy() {
```

Comment:
Nitpick - Intentional?

Reply:
Yes. The returned error is ignored by the method's only consumer.